src/app/src/main/kotlin/org/imperial/mrc/hint/models/ErrorReport.kt

```diff
@@ -0,0 +1,18 @@
+package org.imperial.mrc.hint.models
+
+import java.time.Instant
+
+data class ErrorReport(
+        val email: String,
+        val country: String? = "",
+        val projectName: String,
```

Comment:
Is this field mandatory?

Reply:
We might have a situation where a user sends an error report without uploading baseline data, in that case it should be empty.

Reply:
Sorry, I meant why isn't it like this:
```suggestion
        val projectName: String? = "",
```